### PR TITLE
GUI Styling

### LINKED ISSUE #70: GUI Styling
Creating a more stylish, better looking GUI.

### PR DESCRIPTION
**Pull request by MKerkhof**
Will close #70

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/gui/Style.java [hunk 1]
```diff
@@ -1,9 +1,137 @@
 package nl.tudelft.ti2206.group9.gui;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.net.URL;
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/Style.java]: Unused imports, I believe that's my fault, but they can be removed :)

Comment (MKerkhof) [inline src/main/java/nl/tudelft/ti2206/group9/gui/Style.java]: Didn't notice, will remove them.

File: src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScreen.java [hunk 2]
```diff
@@ -30,28 +32,55 @@ private SettingsScreen() { }
...
             }
         });
 
+        soundButton.setOnAction(new EventHandler<ActionEvent>() {
+            public void handle(ActionEvent event) {
+                if (sound) {
+
+
+                    soundButton.setText("Sound: OFF");
+                    sound = false;
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScreen.java]: Is it just me or are there a lot of blank newlines here? :P

**Comment by mpsijm** [comment]
In my opinion ready to be merged (after my last line comment has been fixed) :)

<fix

File: src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScreen.java (hunk 2 above)
Comment (MKerkhof) [inline src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScreen.java]: Fixed.

<layout

**Comment by thervh70** [comment]
@MKerkhof why is 8bit.ttf included in this commit? It isn't used. Either remove or implement.
If done --> let me know and i'll merge this PR.

< review, organisation

**Comment by mdingjan** [comment]
Apply all of the above and you have also my permission to merge this PR :)

<review

**Comment by thervh70** [comment]
There are enough review comments on this PR, however you should not close this yourself @MKerkhof .

< review

**Pull request merged**